src/ufo-general-backproject-task.c

```diff
@@ -1223,9 +1223,9 @@ copy_to_image (const cl_command_queue cmd_queue,
     cl_event event;
     cl_int errcode;
     cl_mem input_array;
-    const size_t origin[] = {extrema[0], extrema[2], 0};
-    const size_t region[] = {extrema[1] - extrema[0], extrema[3] - extrema[2], 1};
-    size_t src_offset = sizeof (float) * (extrema[2] * width + extrema[0]);
+    const size_t origin[] = {0, extrema[2], 0};
+    const size_t region[] = {width, extrema[3] - extrema[2], 1};
+    size_t src_offset = sizeof (float) * extrema[2] * width;
```

Comment:
No, unfortunately not. I suppose it's that way because supporting that in hardware is probably too much effort because you have to consider all kinds of `cl_image_format`s. A buffer is just bytes ...